lib/IonKey.ts

```diff
@@ -0,0 +1,58 @@
+import { JWK } from 'jose';
+import DidDocumentKey from './DidDocumentKey';
+import JwkEs256k from './models/JwkEs256k';
+import PublicKeyModel from './models/PublicKeyModel';
+import PublicKeyPurpose from './models/PublicKeyPurpose';
+
+/**
+ * Class containing operations related to keys used in ION.
+ */
+export default class IonKey {
```

Comment:
Maybe this is isn't a concern, or maybe it is. I feel like `IonKey` doesn't really mean much? We never used the term `IonKey` anywhere else. I understand it's the key for update and recovery, but I think that's because I have context. What do you think? Would something like `commitmentKey` or `revealKey` make more sense?

Reply:
This is for more than just "operation" keys, this class is for "DID Document" keys also.

The `Ion` prefix in these classes are more of a namespace convention for exported SDK classes.